jmh-core/src/main/java/org/openjdk/jmh/util/Utils.java

```diff
@@ -590,7 +590,22 @@ public static Properties readPropertiesFromCommand(List<String> cmd) {
 
             errDrainer.join();
             outDrainer.join();
-            out.loadFromXML(new ByteArrayInputStream(baos.toByteArray()));
+            try {
+                out.loadFromXML(new ByteArrayInputStream(baos.toByteArray()));
+            } catch (InvalidPropertiesFormatException ex) {
```

Comment:
Move this `catch` block to the existing `try-catch` block. Make the whole thing failing hard without trying to recover. Print the output contents in the exception.